recipes/scdoc/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.43.0"
```

Comment:
```suggestion
required_conan_version = ">=1.33.0"
```

I see no new features which require 1.43 here.

Reply:
corrected